src/confighttp.cpp

```diff
@@ -285,12 +417,23 @@ namespace confighttp {
     response->write(SimpleWeb::StatusCode::success_ok, in, headers);
   }
 
+  /**
+   * @brief Check if a path is a child of another path.
+   * @param base The base path.
+   * @param query The path to check.
+   * @return True if the path is a child of the base path, false otherwise.
+   */
   bool
   isChildPath(fs::path const &base, fs::path const &query) {
     auto relPath = fs::relative(base, query);
     return *(relPath.begin()) != fs::path("..");
   }
 
+  /**
+   * @brief Get an asset from the node_modules directory.
+   * @param response The HTTP response object.
+   * @param request The HTTP request object.
+   */
   void
   getNodeModules(resp_https_t response, req_https_t request) {
```

Comment:
This function returns `bad_request` and could use your function perhaps? In one case it does not return anything so maybe it should also return bad request n that case?